### PR TITLE
Introduce the TimesInMinsAdapted object

### PR DESCRIPTION
**Pull request by shtukas**

### CODE REVIEW

File: common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala [hunk 1]
```diff
@@ -69,6 +69,15 @@ case class TimesInMins(
   cooking: Option[Double]
 )
 
+object TimesInMinsAdapted {
+  def preparationTimeInMinutesFromCuratedRecipe(cr: CuratedRecipe): Double = {
```
Comment (LATaylor-guardian) [inline common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala]: You don't need the `FromCuratedRecipe` part. This is observed from the function parameter.

File: common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala [hunk 2]
```diff
@@ -69,6 +69,15 @@ case class TimesInMins(
...
+    cr.times.preparationHours.getOrElse(0.toDouble) * 60.toDouble + cr.times.preparationMinutes.getOrElse(0.toDouble)
+  }
+  def cookingTimeInMinutesFromCuratedRecipe(cr: CuratedRecipe): Double = {
```
Comment (LATaylor-guardian) [inline common/src/main/scala/com/gu/recipeasy/models/CuratedRecipe.scala]: As above.

**Comment by shtukas** [comment]
You are perfectly right, I am sometimes overly pedantic :)

**Comment by shtukas** [comment]
Done!

**Comment by LATaylor-guardian** [comment]
👍

**Comment by shtukas** [comment]
Thanks!

**Pull request merged**